lib/LambdaHelper.js

```diff
@@ -118,7 +123,7 @@ var _createProxy = function (executionStore, queueInitializedPromise, promise) {
 
         });
 
-        promise.catch(function () {
+        uploadPromise.catch(function () {
             throw "Could not upload function to S3";
```

Comment:
i.e. we should simply console.log the error... eventually we should bubble up those errors as well